add-on/chrome/bootstrap.js

```diff
@@ -1480,10 +1478,22 @@ function shutdown(data, reason) {
   // Stop waiting for browser windows to open.
   wm.removeListener(WindowListener);
 
-  // unregister about: handlers
-  AboutLoop.conversation.unregister(); // eslint-disable-line no-undef
-  AboutLoop.panel.unregister(); // eslint-disable-line no-undef
-  AboutLoop.toc.unregister(); // eslint-disable-line no-undef
+  // XXX do we really want the global PMM here, since it (unnecessarily, I
+  // think) sends a message to the CIPMM (is there any alternative?)
+  let ppmm = Cc["@mozilla.org/parentprocessmessagemanager;1"]
+           .getService(Ci.nsIMessageBroadcaster);
+  ppmm.broadcastAsyncMessage("LoopShuttingDown");
+
+  // XXX should we be listening for a response?  I'm not sure what we'd do if
+  // we got one, so it doesn't really seem worth it
```

Comment:
Well, its an async message, and what can you do about it if it doesn't get there and back? Unless we block on the response, but then we might hang the main process... I think this is just one of the instances where we have to trust it gets there.